specs/slider/slider.md

```diff
@@ -0,0 +1,181 @@
+# Slider + Slider-Label
+
+## Overview
+
+The `slider` component allows the user to move one or two `thumb` elements along a vertical or horizontal axis to select a value or range. `slider` is also considered an input field and must work within a `<form>` as other input fields do.
+
+### Background
+
+Just styling `<input type="range"/>` and using `<datalist>` for the tickmarks was considered but the standards aren't supported enough to allow for the level customization we want to provide.

```

Comment:
Just for clarity, is this that standards are not supported by each browser, or the current standard elements don't support the level of customization we want?

Reply:
each browser has different level of support, no browser seems to have full support for things like the labels. I have no idea how far off they are to implementing this as suggested/documented.

Reply:
Unresolving this - can we add this detail or a bit more here? There is a standard progress element but it doesn't support the full level of detail, styling, etc that most design systems need or require.

Reply:
For context: https://caniuse.com/#feat=input-range

Reply:
I think I resolved this with my updated text (using what you said mostly). matching roughly what is mentioned on atlassian and lightning design slider documentation.